dataprofiler/data_readers/data_utils.py

```diff
@@ -651,22 +651,26 @@ def url_to_bytes(url_as_string, options):
     if 'verify_ssl' in options:
         verify_url = options['verify_ssl']
 
-    with requests.get(url_as_string, stream=True, verify=verify_url) as url:
-        url.raise_for_status()
-        if 'Content-length' in url.headers \
-                and int(url.headers['Content-length']) >= 1024 ** 3:
+    try:
+        with requests.get(url_as_string, stream=True, verify=verify_url) as url:
+            url.raise_for_status()
+            if 'Content-length' in url.headers \
+                    and int(url.headers['Content-length']) >= 1024 ** 3:
 
-            raise ValueError('The downloaded file from the url may not be larger than 1GB')
+                raise ValueError('The downloaded file from the url may not be larger than 1GB')
 
-        total_bytes = 0
-        c_size = 8192
+            total_bytes = 0
+            c_size = 8192
 
-        for chunk in url.iter_content(chunk_size=c_size):
-            stream.write(chunk)
-            total_bytes += c_size
+            for chunk in url.iter_content(chunk_size=c_size):
+                stream.write(chunk)
+                total_bytes += c_size
 
-            if total_bytes > 1024 ** 3:
-                raise ValueError('The downloaded file from the url may not be larger than 1GB')
+                if total_bytes > 1024 ** 3:
+                    raise ValueError('The downloaded file from the url may not be larger than 1GB')
+    except requests.exceptions.SSLError:
+        raise ValueError("The URL given has an untrusted SSL certificate. Although highly discouraged, "
```

Comment:
I think i like the exception thrown by: https://stackoverflow.com/questions/696047/re-raise-exception-with-a-different-type-and-message-preserving-existing-inform
where instead it basically still throws the previous error, so the user can debug it, but also throws its own error:
```python
try:
    ...
except requests.exceptions.SSLError as e:
        raise RuntimeError(.....) from e
```
I chose `RuntimeError` because i think it clarifies it wasn't necessary something wrong with user values, but something happened during runtime and will also still trigger the `SSLError`.
Additionally, i might mention in the description e.g. `options=dict(verify_url=False)`. Not sure this last part is necessary, but may make it more clear.